apps/laboratory/tests/canary.spec.ts

```diff
@@ -6,7 +6,11 @@ const ENV = process.env['ENVIRONMENT'] || 'dev'
 const REGION = process.env['REGION'] || 'eu-central-1'
 
 testConnectedMW.beforeEach(async ({ modalValidator, walletValidator }) => {
+  // eslint-disable-next-line no-console
```

Comment:
FYI you can use `eslint-disable-file no-console` at the top here to avoid needing it on each `console.time` call explicitly: https://stackoverflow.com/questions/34764287/turning-off-eslint-rule-for-a-specific-file

Reply:
Updated to do this